BraintreeUIKit/Helpers/BTUIKAppearance.m

```diff
@@ -123,6 +123,11 @@ + (void)styleSystemLabelSecondary:(UILabel *)label {
     label.textColor = [BTUIKAppearance sharedInstance].secondaryTextColor;
 }
 
++ (void)styleAdjustsToSystemFontSize:(UILabel *)label {
+    label.font = [UIFont preferredFontForTextStyle:UIFontTextStyleBody];
+    label.adjustsFontForContentSizeCategory = YES;
```

Comment:
The `adjustsFontForContentSizeCategory` property was introduced in iOS 10, but drop-in still supports iOS 9, so this will crash on that version of iOS.

Reply:
Ran on iOS 9 simulator and can confirm that this does crash. Making updates.